twill-api/src/main/java/org/apache/twill/api/TwillController.java

```diff
@@ -89,4 +90,29 @@
    * @return A {@link Future} that will be completed when the restart operation has been done.
    */
   Future<String> restartInstances(String runnable, int instanceId, int... moreInstanceIds);
+
+  /**
+   * Set the root log level for Twill applications running in a container.
```

Comment:
In a container? Which container? Isn't that this affect all containers of all runnables as well as the AM?